src/modules/common/components/Tooltip.js

```diff
@@ -0,0 +1,12 @@
+// @flow
+
+import React from 'react'
+import ReactTooltip from 'react-tooltip'
+
+class Tooltip extends React.Component<{}> {
+  render () {
+    return <ReactTooltip effect='solid' delayShow={0} />
+  }
+}
+
+export default Tooltip
```

Comment:
I don't think that we need this file since we should only have one `ReactTooltip` in our renderTree